sim/hunter/kill_shot.go

```diff
@@ -0,0 +1,76 @@
+package hunter
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+func (hunter *Hunter) registerKillShotSpell() {
+	if !hunter.HasRune(proto.HunterRune_RuneLegsKillShot) {
+		return
+	}
+
+	baseDamage := 113 / 100 * 100 / 100 * (2.976264 + 0.641066 * float64(hunter.Level) + 0.022519 * float64(hunter.Level) * float64(hunter.Level)) // taken from wowhead
```

Comment:
You can replace the base calculation that uses the hunter's level with `hunter.baseRuneAbilityDamage()`

Reply:
Also even though the tooltip shows `113 / 100 * 100 / 100 *` we can just do `113 / 100 *`

Reply:
I saw that baseRuneAbilityDamage function at some point and totally didn't realize it was the same!